src/bundle/Resources/public/js/scripts/admin.location.bookmark.js

```diff
@@ -15,14 +16,31 @@
     const isCurrentLocation = (locationId) => {
         return parseInt(locationId, 10) === currentLocationId;
     };
+    const setBookmarkWrapperCheckedClass = (bookmarked) => {
+        const wrapperClassList = document.querySelector(SELECTOR_BOOKMARK_WRAPPER).classList;
+        const bookmarkCheckedClass = 'ez-add-to-bookmarks--checked';
+
+        if (bookmarked) {
+            wrapperClassList.add(bookmarkCheckedClass);
```

Comment:
Not an expert, but I believe there is toggle method
@sunpietro

Reply:
This function also manages CSS class on event: `ez-bookmark-change`. 
I assume that can occur event `ez-bookmark-change` with `bookmarked` property set to `true` even if the content is already bookmarked.

Reply:
You can do it like this `wrapper.classList.toggle('ez-add-to-bookmarks--checked', isBookmarked);`. See the reference: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList